gsplat/relocation.py

```diff
@@ -0,0 +1,63 @@
+import math
+import torch
+from .cuda._wrapper import _make_lazy_cuda_func
+
+N_max = 51
+binoms = torch.zeros((N_max, N_max)).float().cuda()
+for n in range(N_max):
+    for k in range(n + 1):
+        binoms[n, k] = math.comb(n, k)
+
+
+def compute_relocation_cuda(opacity_old, scale_old, N):
```

Comment:
Could you mark the typing of the inputs and the output, and write a couple lines of code docs to explain the  
- what this function is doing / used for
- reference
- input & outputs (shape and meaning)
- warning / notes if there is any

Also we could get rid of the suffix `_cuda`, and add some checking on the inputs to make sure they are:
- cuda tensor
- with the expected shape
- and call `.contiguous()` to make sure the memory layout is continuous.

Reply:
Added. Let me know if I should elaborate more.